pkg/server/routes.go

```diff
@@ -552,4 +541,47 @@ func (s *Server) routes(
 		),
 	)
 	api.Handle("/systems", systemsHandler.Handle())
+
+	// set up GraphQL routes
+	s.router.HandleFunc("/api/playground", playground.Handler("GraphQL playground", "/api/graph/query")) // no need for AuthN, can hang off top router
+	gql := s.router.PathPrefix("/api/graph").Subrouter()
+	gql.Use(authorizationMiddleware) // TODO: see comment at top-level router
+	resolver := graph.NewResolver(
+		store,
+		services.NewFetchSystems(
+			serviceConfig,
+			store.ListSystems,
+			services.NewAuthorizeHasEASiRole(),
+		),
+	)
+	gql.Handle("/query", handler.NewDefaultServer(generated.NewExecutableSchema(generated.Config{Resolvers: resolver})))
+
+	// !! catchall should be the last route added !!
+	s.router.PathPrefix("/").Handler(handlers.NewCatchAllHandler(base).Handle())
+
+	// useful for debugging route issues
+	// s.router.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
+	// 	pathTemplate, err := route.GetPathTemplate()
+	// 	if err == nil {
+	// 		fmt.Println("ROUTE:", pathTemplate)
+	// 	}
+	// 	pathRegexp, err := route.GetPathRegexp()
+	// 	if err == nil {
+	// 		fmt.Println("Path regexp:", pathRegexp)
+	// 	}
+	// 	queriesTemplates, err := route.GetQueriesTemplates()
+	// 	if err == nil {
+	// 		fmt.Println("Queries templates:", strings.Join(queriesTemplates, ","))
+	// 	}
+	// 	queriesRegexps, err := route.GetQueriesRegexp()
+	// 	if err == nil {
+	// 		fmt.Println("Queries regexps:", strings.Join(queriesRegexps, ","))
+	// 	}
+	// 	methods, err := route.GetMethods()
+	// 	if err == nil {
+	// 		fmt.Println("Methods:", strings.Join(methods, ","))
+	// 	}
+	// 	fmt.Println()
+	// 	return nil
+	// })
```

Comment:
I'm usually not one for leaving commented out code in the codebase (because you can just "hide" it in an earlier check-in), but this seemed useful/helpful for when people are mucking about with routes and having a hard time understanding what's going wrong.

Reply:
I have often wanted this output from the app, so thank you for implementing this. I wonder if we could make it something that the `easi` executable prints out when invoked with `--print-routes`?

Reply:
Could also be a feature flag? Though that might be overkill